src/lib/transition_frontier/persistent_frontier/persistent_frontier.ml

```diff
@@ -182,7 +183,8 @@ module Instance = struct
       Error `Bootstrap_required )
 
   let load_full_frontier t ~root_ledger ~consensus_local_state ~max_length
-      ~ignore_consensus_local_state ~precomputed_values =
+      ~ignore_consensus_local_state ~precomputed_values
```

Comment:
Can we check in this function that longest chain is 290 blocks if chain length >= 290?

Reply:
We would not always have the longest chain be of length k, so I don't think this check would make sense here.
For example, in bootstrap, the longest chain would be of length 1;
or if a node restarted shortly after bootstrap, then its longest chain would have length less than k.

The persistent frontier currently doesn't have the ability to track the state of the node, so this check is not so reasonable for the current implementation.